lantern/src/main/java/github/nisrulz/lantern/Lantern.java

```diff
@@ -82,6 +82,10 @@ public void turnOffFlashlight(Context context) {
       isFlashOn = false;
     }
   }
+  
+  public boolean isFlashOn() {
```

Comment:
Should it be called `is*Flashlight*On()` if the other methods use *Flashlight* instead of *Flash*? E.g. `turnFlashlightOn()` and `turnFlashlightOff()`